### PR TITLE
feat(recipient_app): added json serializer

### PR DESCRIPTION
**Pull request by novas1r1**
I've added the json_serializable package to have automated json_serialization to remove some unnecessary code which can be automated easily :)

Merge https://github.com/socialincome-san/public/pull/426 first!

### CODE REVIEW

**Comment by MDikkii** [comment]
Should it have base branch as `387`?

File: recipients_app/lib/core/helpers/date_time_converter.dart [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import "package:json_annotation/json_annotation.dart";
+
+class DateTimeConverter implements JsonConverter<DateTime, String> {
+  const DateTimeConverter();
+
+  @override
+  DateTime fromJson(String json) =>
+      DateTime.fromMillisecondsSinceEpoch(int.parse(json));
```
[the remark this answers is not in the document]
Comment (novas1r1) [reply recipients_app/lib/core/helpers/date_time_converter.dart:8]: We do have the timestamp converter :)

[the remark this answers is not in the document]
Comment (novas1r1) [reply recipients_app/lib/core/helpers/date_time_converter.dart:8]: the recipient_since field was somehow a datetime field. I don't know why? I just took it over. If you know something about that, we can remove the dateTime convertert

[the remark this answers is not in the document]
Comment (MDikkii) [reply recipients_app/lib/core/helpers/date_time_converter.dart:8]: TBH I thought that Timestamp is the Firestore data type and DateTime is the dart data type 🤷

**Pull request merged**